lms/djangoapps/teams/tests/test_csv.py

```diff
@@ -241,14 +243,24 @@ def setUpClass(cls):
                 'id': 'teamset_1',
                 'name': 'teamset_name',
                 'description': 'teamset_desc',
+                'max_team_size': 3,
             }]
         })
         cls.course = CourseFactory(teams_configuration=teams_config)
         cls.second_course = CourseFactory(teams_configuration=teams_config)
+        cls.import_manager = None
+
+    def setUp(self):
+        """ Initialize import manager """
+        super(TeamMembershipImportManagerTests, self).setUp()
+        self.import_manager = csv.TeamMembershipImportManager(self.course)
+        self.import_manager.teamset_ids = {ts.teamset_id for ts in self.course.teamsets}
 
-        # initialize import manager
-        cls.import_manager = csv.TeamMembershipImportManager(cls.course)
-        cls.import_manager.teamset_ids = {ts.teamset_id for ts in cls.course.teamsets}
+    def tearDown(self):
```

Comment:
am I seeing something weird with diffs? I though you removed this? is it required?

Reply:
I thought you were just talking about the `super()` call. Removed.